app/build.gradle

```diff
@@ -20,7 +19,15 @@ android {
         buildConfigField("String", "web_client_id", keyProperties['web_client_id'])
     }
     buildTypes {
+        debug {
+            buildConfigField("String", "BACKEND_URI", "\"pear-dev.cornellappdev.com\"")
+            manifestPlaceholders = [usesCleartextTraffic: "true"]
```

Comment:
Do we need to downgrade to http for dev purposes?

Reply:
Yeah, I occasionally use a local backend for testing.